### DISCUSSION TITLE
Please add the update skipping or update suppression switch

### DISCUSSION THREAD

**Discussion by Walkersifolia** [PurpurMC/Purpur#1373] [Feature Requests]
Some redstone player need to cut the nether portal, and it will use the update skipping, but in latest version of purpur, the update skipping is disabled. I wish to add a switch in `purpur.yml` to make the update skipping and OutOfMemory suppression work again. The default value can be `false`, and if I want to use it, I need to turn it to `true`

**Comment by CatTeaA** [comment]
1.20 has fixed update skipping, which is also what mojang means

**Reply by Walkersifolia** [reply]
I have tried the update skipper can work in the 1.20.1 vanilla and fabric server.

**Reply by CatTeaA** [reply]
Good for you

**Reply by Walkersifolia** [reply]
But it can't work at paper or purpur or spigot server
